#! /bin/sh /usr/share/dpatch/dpatch-run
## install_scripts_config.dpatch by Ralf Treinen <treinen@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use for installatation a patched config.sh (created by debian/rules)
## DP: Ralf Treinen <treinen@debian.org>

@DPATCH@
diff -urNad ocaml~/build/install.sh ocaml/build/install.sh
--- ocaml~/build/install.sh	2008-10-17 12:22:50.000000000 +0200
+++ ocaml/build/install.sh	2008-10-20 15:01:11.000000000 +0200
@@ -18,7 +18,7 @@
 
 cd `dirname $0`/..
 
-. config/config.sh
+. config/config.debian.install.sh
 
 not_installed=$PWD/_build/not_installed
 
diff -urNad ocaml~/build/partial-install.sh ocaml/build/partial-install.sh
--- ocaml~/build/partial-install.sh	2008-10-17 12:22:50.000000000 +0200
+++ ocaml/build/partial-install.sh	2008-10-20 15:01:11.000000000 +0200
@@ -22,7 +22,7 @@
 
 cd `dirname $0`/..
 
-. config/config.sh
+. config/config.debian.install.sh
 
 not_installed=$PWD/_build/not_installed
 
